CHANGELOG.md

```diff
@@ -1,5 +1,57 @@
 # Changelog
 
+## 0.0.1-alpha.12
+
+### Bug fixes
+
+- Avoid duplicate diagnostic when reporting errors in unpacked assignments ([#18897](https://github.com/astral-sh/ruff/pull/18897))
+- Fix panics when "pulling types" for `ClassVar` or `Final` parameterized with >1 argument ([#18824](https://github.com/astral-sh/ruff/pull/18824)). These could cause issues when hovering over symbols in an IDE.
+
+### Improved modeling of Python runtime semantics
+
+- Add support for `@staticmethod`s ([#18809](https://github.com/astral-sh/ruff/pull/18809))
+- Assignments in the body of a staticmethod are never instance attributes ([#18587](https://github.com/astral-sh/ruff/pull/18587))
```

Comment:
This PR also adds the ability to understand pure class attributes that are assigned in `@classmethod`s. This is probably the more interesting feature, the staticmethod thing is just a bugfix (that probably didn't affect anyone). So maybe
```suggestion
- Discover implicit class attribute assignments in `@classmethod`s ([#18587](https://github.com/astral-sh/ruff/pull/18587))
```

Reply:
thanks -- I think it is good to mention the staticmethod change, but I agree the classmethod change is more important! I applied a variant of this in https://github.com/astral-sh/ty/pull/700/commits/12a4194260759e4cb7fab128d400ea0ea94269ab